app/src/main/java/org/eyeseetea/malariacare/domain/usecase/SaveSurveyAnsweredRatioUseCase.java

```diff
@@ -17,6 +17,16 @@ public SaveSurveyAnsweredRatioUseCase(
 
     public void execute(long idSurvey, GetSurveyAnsweredRatioUseCase.Callback callback) {
```

Comment:
This usecase should use AsyncExecutor to save on database and MainExecutor to invoke callbacks